frontend/src/components/Navigator/components/SideNav.js

```diff
@@ -62,7 +62,7 @@ function SideNav({
   ));
 
   return (
-    <Sticky top={50} enabled={!isMobile}>
+    <Sticky className="smart-hub-sidenav" top={50} enabled={!isMobile}>
```

Comment:
Yeah, I had created BEM style identifiers for a few elements, using the same naming conventions as the USWDS (e.g. 'usa-card'). Originally, I planned to style those elements using those classes, but ended up not needing to. I left them in as identifiers since I could see a need to adjust those elements in the future.